bin/sh.VisualElementsManifest.xml

```diff
@@ -0,0 +1,6 @@
+<Application xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<VisualElements
+		BackgroundColor="#F0EFE7"
```

Comment:
Where does this color code come from, i.e. why is it not pure white (`#FFFFFF`)?